src/rust/rust-proto/src/graplinc/grapl/api/plugin_registry/v1beta1.rs

```diff
@@ -364,6 +367,236 @@ impl ProtobufSerializable for DeployPluginResponse {
     type ProtobufMessage = proto::DeployPluginResponse;
 }
 
+//
+// PluginDeployment
+//
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PluginDeploymentStatus {
+    Unspecified,
```

Comment:
i'd say that we should probably just follow the example of other rust-proto enums where Unspecified is never really expected to be handled at runtime? but i'm not committed to that